Topics/Tech_Stacks/Angular Basics.md

```diff
@@ -0,0 +1,195 @@
+
+# Understanding Angular: A Guide for Beginners
+
+# Prerequisites
+- HTML
+- CSS
+- TypeScript
+
+## 1. What are Components?
+
+### Definition
+In Angular, components are the fundamental building blocks. They control a part of the UI for your application. Each component consists of three main parts:
+
+- **Template**: This is the HTML part, defining how the component looks.
+- **Class**: Written in TypeScript, it handles the data and behavior of the component.
+- **Decorator**: This provides metadata about the component, like its selector and template URL.
+
+For example
+
+**list.component.ts** (files are stored as .ts, guess what ts stands for - hint: it's not Top Secret)
+```
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-list',
+  template: `list.component.html`,
+})
+export class ListComponent {
+    users = {
```

Comment:
Just checking, does this need to be a square bracket?

Reply:
Yes, it should be. Thanks for pointing it out!